src/app/ui/ResultExpertInfo.jsx

```diff
@@ -0,0 +1,65 @@
+import mean from 'mean'
+import variance from 'variance'
+import React from 'react'
+import { createSelector, createStructuredSelector } from 'reselect'
+
+import getNonMissedDeltas from '../interactors/getNonMissedDeltas'
+
+export default class ResultExpertInfo extends React.Component {
+  static propTypes = {
+    deltas: React.PropTypes.array,
+    meticulousWindow: React.PropTypes.number,
+    preciseWindow: React.PropTypes.number,
+    noteCount: React.PropTypes.number
+  }
+  getStats = (() => {
+    const selectExpertScore = createSelector(
+      (props) => props.deltas,
+      (props) => props.meticulousWindow,
+      (props) => props.preciseWindow,
+      (deltas, meticulousWindow, preciseWindow) => {
```

Comment:
Extract this logic to getLR2ExpertScore so that it can be reused for stat-keeping.